### ISSUE TITLE
Support Facade fixes like \Auth, \Log

### ISSUE THREAD

**Issue by rognales** [imanghafoori1/laravel-microscope#238]
Hi There,

Been using your package happily for the past years. Thanks alot!

I wonder if you can support removing \Auth and replacing with proper imports

**Comment by imanghafoori1** [comment]
Yeah, that is possible but takes time. Currently, I am very busy these days.
My main focus is to stabilize and edebug the previous features. plus php8.2 is coming out and the syntax must be supported.
